app/views/pages/get-started/contact-us-tutorial/building-the-ui.liquid

```diff
@@ -0,0 +1,99 @@
+
+#### app/views/pages/index.html.liquid
+
+{% raw %}
+```liquid
+<h2>Contact Us</h2>
+<form>
+    <div>
+        <label for="email">Email</label>
+        <input type="text" name="contact[email]" id="email">
+    </div>
+<div>
+      <textarea name="contact[body]"></textarea>
+    </div>
+    <input type="submit" value="Send">
+  </form>
+```
+{% endraw %}
+
+You might notice that we used `index.html.liquid` for our homepage file instead of `index.html`. This is because our tool can download websites and store them in HTML format, which can still be rendered by platformOS. Although using `.liquid` is not strictly necessary, we recommend it for consistency and ease of use.
+
+{% include 'alert/tip', content: 'The file extension `.html.liquid` specifies HTML as a `format` and Liquid as a render engine, allowing the use of both HTML and Liquid templating. This makes it easy to render dynamic content using platformOS. Read more about [formats](/developer-guide/pages/pages#formats).' %}
+
+## Using slug vs path to the file
+
+While it's possible to add a slug to the page that holds the homepage content, it's not the recommended approach, unless necessary. If someone needs to modify the homepage later, finding the correct file can be challenging if it’s not clearly named. For example, if the homepage content is in a file named `hello.html`, it’s not immediately clear that this file represents the homepage. This can be even more confusing if there are nested directories.
+
+The recommended approach is to use the file path as the default route. This method automatically uses the name of the file as the path, which greatly improves user experience and maintainability. 
+
+The name of the file will be automatically used as the path. In our case, `app/views/pages/index.html.liquid` will default to `/`.
+
+However, you will see some exceptions later in this task, such as creating the `/contacts/create` page using a slug `contacts` to follow RESTful URL conventions. For more information on using RESTful URLs, [refer to this guide](https://apiguide.readthedocs.io/en/latest/build_and_publish/use_RESTful_urls.html).
+
+{% include 'alert/tip', content: '
+  The homepage slug is `/`, which will work for 
+  <ul>
+    <li><a href="https://example.com">https://example.com</a></li>
+    <li><a href="https://example.com/">https://example.com/</a></li>
+    <li><a href="https://example.com/index.html">https://example.com/index.html</a></li>
+  </ul>
+  The slug of the file `app/views/pages/index.liquid` defaults to `/`.
+' %}
+
+## Deploy and check
+
+Deploy your file
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+pos-cli deploy staging
+</code></pre>
+
+or sync your changes to the instance:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+pos-cli sync staging
+</code></pre>
```

Comment:
Deploying will work after the fact, that is true, but syncing requires us to run the sync before saving the file. In the previous step we already had run the `sync` command so it feels redundant here.

Reply:
You're right. I modified the text.